src/neo-vm/InteropService.cs

```diff
@@ -17,12 +17,15 @@ public InteropService()
 
         protected void Register(string method, Func<ExecutionEngine, bool> handler)
         {
-            dictionary[method] = handler;
+            dictionary[method.ToInteropMethodHash()] = handler;
```

Comment:
Sorry for this delay.
Maybe we should register both, for speed up the current deployed contracts?